### PR TITLE
.github/workflows: add paths for github actions

### LINKED ISSUE #54: add paths for github actions

### PR DESCRIPTION
**Pull request by Hulongyin**
This PR is to fix #54 , add paths for github actions

### CODE REVIEW

**Comment by leyao-daily** [comment]
Not sure why we need add the path definition, we may need it to run on all the repo files?

File: .github/workflows/pr-pylint.yaml [hunk 1]
```diff
@@ -4,7 +4,11 @@ on:
   push:
     branches:
     - main
+    paths:
+    - 'cnap/**/**.py'
```
Comment (leyao-daily) [inline .github/workflows/pr-pylint.yaml]: Maybe **/*.py instead of **/**.py ?

**Comment by Hulongyin** [comment]
> Not sure why we need add the path definition, we may need it to run on all the repo files?

To avoid triggering all actions for each PR, for example, the PR for document updating should not trigger the python code scan action.

File: .github/workflows/pr-license-check.yaml [hunk 2]
```diff
@@ -4,7 +4,11 @@ on:
   push:
     branches:
     - main
+    paths:
+    - 'cnap/**/requirements.txt'
   pull_request:
+    paths:
+    - 'cnap/**/requirements.txt'
```
Comment (hwang37) [inline .github/workflows/pr-license-check.yaml:11]: Only check requirements.txt license?
Shouldn't we check all source code?

Comment (Hulongyin) [reply .github/workflows/pr-license-check.yaml:11]: For license check, we use [liccheck](https://pypi.org/project/liccheck/) to check python packages listed in `requirement.txt` file, so we don't need to check all source code in license check.

File: .github/workflows/pr-pylint.yaml (hunk 1 above)
Comment (Hulongyin) [reply .github/workflows/pr-pylint.yaml]: Thanks, fixed.

**Review by leyao-daily — APPROVED** [review]
LGTM.

**Pull request merged**